samples/contrib/versioned-pipeline-ci-samples/mnist-ci-sample/README.md

```diff
@@ -0,0 +1,25 @@
+# Mnist CI Pipeline
```

Comment:
The first occurrence of  a terminology better be fully spelled out. Continuous Integration (CI)

Reply:
OK.